mautic/mautic
# Improving README.md

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR improves the repository's readme file to be more user-friendly.
Also:
- Adds new stats for repository activity
- Updates the link for dev docs
- Adds links for quickly get started (issues for new devs to fix and best PRs for testing)
- Adds links for the Mautic Trials
- Adds a list of benefits
- Centralizes help resources under one heading
- Converts the disclaimer into a warning
- Reorganizes the contribution section






---
### 📋 Steps to test this PR:


1. Open the Files changed tab
2. Find the readme file
3. Click on options (3 dots) and select View file
4. On the new tab opened, check the markdown structure
5. Go back and check the logo file (it will appear broken when viwing the readme file because the file doesn't exist in the repository yet)
